docker/hub-tool
# fix the error prints on normal command cancelation

**- What I did**
Fixed the error prints in case of normal cancelation
closes #54 

**- How I did it**
Exported the cancel error and add a check when the cobra command run handler

**- How to verify it**
Run one of the commands e.g. `hub-tool login` and instead of entering Username as expected just press ^C.
No print like those should appear:
`Username: ^CError: canceled`

**- Description for the changelog**

Fix graceful shutdown handling

**- A picture of a cute animal (not mandatory)**
